src/server/decommission-service/controllers/decommission-form.js

```diff
@@ -0,0 +1,27 @@
+import { buildOptions } from '~/src/server/common/helpers/options/build-options.js'
+import { fetchRepositories } from '~/src/server/common/helpers/fetch/fetch-repositories.js'
+
+const decommissionFormController = {
+  handler: async (request, h) => {
+    const query = request?.query
+    const redirectLocation = query?.redirectLocation
+
+    const { repositories } = await fetchRepositories()
+
+    const repositoriesValues = repositories.map((repo) => {
+      return { value: repo.id, text: repo.description }
+    })
+
+    const repositoriesOptions = buildOptions(repositoriesValues ?? [])
+
+    return h.view('decommission-service/views/decommission-form', {
+      pageTitle: 'Decommission Service',
+      heading: 'Decommission a service',
+      headingCaption: 'Provide the service name to decommission',
+      redirectLocation,
```

Comment:
not sure why this is being passed around? I suspect this is leftovers from the multi step form work. I don't think you need to pass the redirect location along in the forms here? it doesn't seem to doing anything